### ISSUE TITLE
No Horizontal Scrollbar with Columns

### ISSUE THREAD

**Issue by jgarethw** [swistakm/pyimgui#151]
Hi there,
I've run into an issue with using Columns, I can't get the horizontal scrollbar to work with it.
The demo C++ code in imgui shows this:

```c++
        ImGui::SetNextWindowContentSize(ImVec2(1500.0f, 0.0f));
        ImVec2 child_size = ImVec2(0, ImGui::GetFontSize() * 20.0f);
        ImGui::BeginChild("##ScrollingRegion", child_size, false, ImGuiWindowFlags_HorizontalScrollbar);
        ImGui::Columns(10);
        int ITEMS_COUNT = 2000;
        ImGuiListClipper clipper(ITEMS_COUNT);  // Also demonstrate using the clipper for large list
        while (clipper.Step())
        {
            for (int i = clipper.DisplayStart; i < clipper.DisplayEnd; i++)
                for (int j = 0; j < 10; j++)
                {
                    ImGui::Text("Line %d Column %d...", i, j);
                    ImGui::NextColumn();
                }
        }
        ImGui::Columns(1);
        ImGui::EndChild();
```
I don't think the SetNextWindowContentSize is implemented in pyimgui so I tried this code:

```python
	imgui.set_next_window_size(1500, 0)
        child_flags = imgui.WINDOW_HORIZONTAL_SCROLLING_BAR | imgui.WINDOW_NO_SAVED_SETTINGS
        imgui.begin_child("child_region", 500, 500, border=False, flags=child_flags)
        imgui.columns(10)
        imgui.set_column_width(0, 100)
        imgui.set_column_width(1, 100)
        imgui.set_column_width(2, 100)
        imgui.set_column_width(3, 100)
        imgui.set_column_width(4, 100)
        imgui.set_column_width(5, 100)
        imgui.set_column_width(6, 100)
        imgui.set_column_width(7, 100)
        imgui.set_column_width(8, 100)
        imgui.set_column_width(9, 100)

        for i in range(100):
            for j in range(10):
                imgui.text("Line " + str(i) + " Column " + str(j) + "...")
                imgui.next_column()
            imgui.separator()

        imgui.columns(1)
        imgui.end_child()
```

which results in a grid with vertical scrollbar but no horizontal scrollbar even though the columns extend off the child region:
![columns](https://user-images.githubusercontent.com/10236994/84265316-94131780-ab1a-11ea-8b29-f4d327123490.PNG)

Is there a way to make this work in the current version of pyimgui? many thanks in advance!

**Comment by swistakm** [comment]
You're right. `SetNextWindowContentSize` is missing and that is the function you should use to have horizontal scrollbars in your use case. This was already implemented by @Red-Pheonix in #143 but I did not have enough time to make official release since that time. Anyway, I'm on it and it is a matter of days.

**Comment by swistakm** [comment]
Feature that allows to resolve that issue released in 1.2.0
